java/src/main/java/com/saucelabs/simplesauce/Options.java

```diff
@@ -0,0 +1,274 @@
+package com.saucelabs.simplesauce;
+
+import lombok.Getter;
+
+import java.util.*;
+
+public class Options {
+    private static final class BrowserLookup {
+        private static final Map<String, String> lookup = new HashMap<String, String>();
+    }
+
+    private static final class PlatformLookup {
+        private static final Map<String, String> lookup = new HashMap<String, String>();
+    }
+
+    private static final class PageLoadStrategyLookup {
+        private static final Map<String, String> lookup = new HashMap<String, String>();
+    }
+
+    private static final class TimeoutsLookup {
+        private static final Map<String, String> lookup = new HashMap<String, String>();
+    }
+
+    private static final class UnhandledPromptBehaviorLookup {
+        private static final Map<String, String> lookup = new HashMap<String, String>();
+    }
+
+    private static final class JobVisibilityLookup {
+        private static final Map<String, String> lookup = new HashMap<String, String>();
+    }
+
+    private static final class PreRunLookup {
+        private static final Map<String, String> lookup = new HashMap<String, String>();
+    }
+
+    public enum PageLoadStrategy {
+        NONE("none"),
+        EAGER("eager"),
+        NORMAL("normal");
+
+        @Getter private String value;
+
+        public static Set keys() {
+            return PageLoadStrategyLookup.lookup.keySet();
+        }
+
+        PageLoadStrategy(String value) {
+            this.value = value;
+            PageLoadStrategyLookup.lookup.put(value.toString(), this.name());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid calling toString() on String objects; this is unnecessary.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4758011)